test/test.js

```diff
@@ -1,21 +1,27 @@
 var assert = require('assert')
 var getStdin = require('get-stdin')
 var got = require('got')
+var jsapi = require('../src/hastebin')
 
-getStdin()
+var testData = 'test'
 
-.then(function (stdin) {
-  var hastebinUrl = stdin.trim()
-  assert(hastebinUrl.match(/^https:\/\/hastebin.com\/.*/) != null, 'stout must be a hastebin url')
+function testOutput (input) {
```

Comment:
Can you rename this to something more descriptive like `assertHastebinUrlMatchesTestData` or `assertHastebinUrlMatchesText`

Reply:
Done